spyder/plugins/editor/lsp/providers/workspace.py

```diff
@@ -0,0 +1,178 @@
+# -*- coding: utf-8 -*-
+
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Spyder Language Server Protocol Client workspace handler routines."""
+
+import logging
+
+from spyder.plugins.editor.lsp.providers.utils import (
+    path_as_uri, process_uri, match_path_to_folder)
+from spyder.plugins.editor.lsp import LSPRequestTypes, ClientConstants
+from spyder.plugins.editor.lsp.decorators import (
+    handles, send_request, send_response, send_notification)
+
+logger = logging.getLogger(__name__)
+
+
+class WorkspaceProvider:
+    @send_notification(method=LSPRequestTypes.WORKSPACE_CONFIGURATION_CHANGE)
+    def send_plugin_configurations(self, configurations, *args):
+        self.plugin_configurations = configurations
+        params = {
+            'settings': configurations
+        }
+        return params
+
+    @handles(LSPRequestTypes.WORKSPACE_FOLDERS)
+    @send_response
```

Comment:
Please change the order of these decorators to make text easier to read, like this:

```python
@send_response
@handles(LSPRequestTypes.WORKSPACE_FOLDERS)
```